unit/src/GenericRealUTest.C

```diff
@@ -0,0 +1,177 @@
+//* This file is part of the MOOSE framework
```

Comment:
I would like to see an *.i test of this capability too.

Reply:
I plan to let LAROMANCE stressupdate use this type, which is tested. (This is still a draft PR!)